src/DotNet/DotNet.csproj

```diff
@@ -3,20 +3,25 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>false</IsPackable>
     <InstallWorkloadPacks Condition=" '$(InstallWorkloadPacks)' == '' ">true</InstallWorkloadPacks>
+    <InternalAzureFeed>https://dotnetbuilds.blob.core.windows.net/internal</InternalAzureFeed>
     <DotNetFeedUrl>https://dotnetbuilds.blob.core.windows.net/public</DotNetFeedUrl>
   </PropertyGroup>
   <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('windows'))">
     <DotNetInstallScriptUrl>https://dot.net/v1/dotnet-install.ps1</DotNetInstallScriptUrl>
     <DotNetInstallScriptName>dotnet-install.ps1</DotNetInstallScriptName>
     <DotNetInstallScriptPath>$(DotNetOutputPath)$(DotNetInstallScriptName)</DotNetInstallScriptPath>
-    <DotNetInstallCommand>&amp; '$(DotNetInstallScriptPath)' -Version $(MicrosoftDotnetSdkInternalPackageVersion) -InstallDir '$(DotNetDirectory)' -AzureFeed $(DotNetFeedUrl) -Verbose</DotNetInstallCommand>
-    <DotNetInstallCommand>powershell -ExecutionPolicy ByPass -NoProfile -Command &quot;$(DotNetInstallCommand)&quot;</DotNetInstallCommand>
```

Comment:
Why was `-ExecutionPolicy ByPass -NoProfile` removed here? I can no longer build the repo on my machine:

```
  Downloading from "https://dot.net/v1/dotnet-install.ps1" to "C:\git\maui\bin\dotnet-install.ps1" (65,633 bytes).
  & : File C:\git\maui\bin\dotnet-install.ps1 cannot be loaded because running scripts is disabled on this system. For
  more information, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
  At line:1 char:3
  + & 'C:\git\maui\bin/dotnet-install.ps1' -Version 6.0.200-rtm.22107.3 - ...
  +   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : SecurityError: (:) [], PSSecurityException
      + FullyQualifiedErrorId : UnauthorizedAccess
C:\git\maui\src\DotNet\DotNet.csproj(104,5): error MSB3073: The command "powershell -Command "& 'C:\git\maui\bin/dotnet-install.ps1' -Version 6.0.200-rtm.22107.3 -InstallDir 'C:\git\maui\bin/dotnet/' -Verbose"" exited with code 1.
```

See https://github.com/dotnet/maui/pull/4295

Reply:
It's probably an accident, @rmarinho can you put these back?

Reply:
yeah it as accident, i saw the PR adding those in i will review and merge